app/javascript/components/rooms/room/shared_access/forms/SharedAccessForm.jsx

```diff
@@ -42,8 +42,8 @@ export default function SharedAccessForm({ handleClose }) {
                       <Stack direction="horizontal" className="py-2">
                         <Form.Check
                           type="checkbox"
+                          key={user.id}
```

Comment:
What's the use of `key` here?

Reply:
I read that it is recommended to use the key attributes inside arrays
https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Yes, it should be done on the parent component which you have done on #L39 `<tr key={user.id} className="align-middle">`,
The key on the `Form.Check` I believe isn't requireid,